lib/ansible_spec/load_ansible.rb

```diff
@@ -306,33 +326,15 @@ def self.get_variables(host, group_idx)
     end
 
     # all group
-    vars_file = 'group_vars/all.yml'
-    if File.exist?(vars_file)
-      yaml = YAML.load_file(vars_file)
-      if yaml.kind_of?(Hash)
-        vars.merge!(yaml)
-      end
-    end
+    load_vars_file(vars ,'group_vars/all')
```

Comment:
if you can change for the above comment, Could you modify this line?
example:
`vars = load_vars_file(vars ,'group_vars/all')`